### PR TITLE
Great job dude!!! adding cargo project and add github actions

### PR DESCRIPTION
**Pull request by darioalessandro**
I wrapped your merge_sort file into a cargo project, then added github action just for laughs and giggles. 

Let me know if this is helpful 😄

### CODE REVIEW

File: src/merge_sort.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[test]
+    fn test_merge_sort() {
+        let v = vec![9, 6, 3, 16, 25, 2, 1, 6, 1239, 3, 1969, 20, 2];
+        let result = merge_sort(v.clone());
+        assert_eq!(result, vec![1, 2, 2, 3, 3, 6, 6, 9, 16, 20, 25, 1239, 1969])
```
Comment (darioalessandro) [inline src/merge_sort.rs:36]: @snk-js I fixed the test, there was only 1 `3` in the assertion 😄

File: src/merge_sort.rs [hunk 2]
```diff
@@ -0,0 +1,38 @@
+use core::fmt::Debug;
```
Comment (darioalessandro) [inline src/merge_sort.rs:1]: Added missing import

**Comment by snk-js** [comment]
Amazing Dario! I'm so happy with your effort on helping me all of sudden, I will update more on Rust as I am focused to learn and someday to work with it. Thanks, it's pretty motivating.

**Comment by snk-js** [comment]
I closed instead of merging lol 
<iframe src="https://giphy.com/embed/TVdjQN5o31JTi" width="440" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/jump-dumb-TVdjQN5o31JTi">via GIPHY</a></p>

**Comment by snk-js** [comment]
I closed instead of merging lol
https://giphy.com/gifs/jump-dumb-TVdjQN5o31JTi

**Comment by snk-js** [comment]
I closed instead of merging lol
https://media0.giphy.com/media/TVdjQN5o31JTi/giphy.gif?cid=790b761171e9bdf1725618103c12d617fb4073f1a1cf1ff0&rid=giphy.gif&ct=g

**Comment by snk-js** [comment]
I closed instead of merging lol, sorry

stupidity

**Comment by darioalessandro** [comment]
@snk-js glad you liked it and to hear that you are still interested in rust. 🤗🤗

**Pull request merged**